plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/KtlintCheckTask.kt

```diff
@@ -86,6 +86,7 @@ open class KtlintCheckTask @Inject constructor(
     @TaskAction
     fun lint() {
         checkMinimalSupportedKtlintVersion()
+        checkCWEKtlintVersion()
```

Comment:
This is a vulnerability in both the `check` and the `lint` tasks. I want to make sure this is applied to both plugins.

Reply:
Yes, as format extends check task - it works for both.